pkgs/development/libraries/libgda/default.nix

```diff
@@ -7,7 +7,8 @@
 assert mysqlSupport -> libmysqlclient != null;
 assert postgresSupport -> postgresql != null;
 
-(if stdenv.isAarch64 then overrideCC stdenv gcc6 else stdenv).mkDerivation rec {
+# (if stdenv.isAarch64 then overrideCC stdenv gcc6 else stdenv).mkDerivation rec {
```

Comment:
I suspect this is what is causing it to be unsupported? I made this change after adding `{ allowUnsupportedSystem = true; }` to `~/.config/nixpkgs/config.nix`. I want to rebuild the package without that config.nix change so we can see if that is the problem.
See https://github.com/NixOS/nixpkgs/commit/2c1e736ae8ea308cecc6b6f2cf44638280393363

Reply:
Nevermind, I still needs the config.nix change. I will revert this hunk